platform/server/elasticsearch/ElasticsearchFacade.ts

```diff
@@ -7,6 +7,11 @@ import { Cluster } from './Cluster';
 export class ElasticsearchRequestHelpers {
   constructor(private readonly elasticsearchService: ElasticsearchService) {}
 
+  // NB: now unused. This was getting bound to
+  // to the router's `onRequest` method, which has
+  // been abandoned in favor of creating services
+  // (for plugins) in the request handler or plugin
+  // constructor
```

Comment:
Rip this out? Leave it? Maybe keep `ElasticsearchFacade` and decorate `ElasticsearchService` with it? Might be a nice way to separate concerns.

Reply:
The idea behind this was having a nice api for getting the current value. But I think we can solve that later by just having a helper along the lines of:

```js
const latest = (obs) => k$(obs)(first(), toPromise());
```

I think adding these facade things now just clutters things, so I like just having it on the service (where you added `getScopedCluster`)

Reply:
Ok, ripping.